[Bug][CI] All unit tests execute failed

### What happened

During execution of the unit test job, the task `Set up swapfile path` failed with the following error:

```
vm.max_map_count = 262144
fs.file-max = 65536
fallocate: fallocate failed: Text file busy
Error: Process completed with exit code 1.
```

### What you expected to happen

Unit test job should not failed.

### How to reproduce

See this job: https://github.com/apache/inlong/actions/runs/22342711119/job/64649510691?pr=12074

### Environment

_No response_

### InLong version

master

### InLong Component

Other for not specified component

### Are you willing to submit PR?

- [x] Yes, I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/.github/workflows/ci_ut.yml
+++ b/.github/workflows/ci_ut.yml
@@ -109,6 +109,15 @@ jobs:
         run: |
           sudo sysctl -w vm.max_map_count=262144
           sudo sysctl -w fs.file-max=65536
+          if swapon --show=NAME | grep -q '^/swapfile$'; then
+            echo "Swapfile already active, skipping creation"
+            exit 0
+          fi
+          if [ -f /swapfile ]; then
+            echo "Swapfile exists but not active, recreating"
+            sudo swapoff /swapfile || true
+            sudo rm -f /swapfile
+          fi
           sudo fallocate -l 5G /swapfile
           sudo chmod 600 /swapfile
           sudo mkswap /swapfile

